<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>TripListDialog</class>
 <widget class="QDialog" name="TripListDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>550</width>
    <height>800</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>550</width>
    <height>800</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>550</width>
    <height>800</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>我的行程</string>
  </property>
  <widget class="QLabel" name="backgroundLabel">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>550</width>
     <height>800</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>550</width>
     <height>800</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>550</width>
     <height>800</height>
    </size>
   </property>
   <property name="styleSheet">
    <string notr="true">background-color: rgb(255, 255, 255);</string>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QLabel" name="label">
   <property name="geometry">
    <rect>
     <x>21</x>
     <y>30</y>
     <width>121</width>
     <height>51</height>
    </rect>
   </property>
   <property name="styleSheet">
    <string notr="true">font: 75 13pt &quot;Adobe Arabic&quot;;</string>
   </property>
   <property name="text">
    <string>我的行程</string>
   </property>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>110</y>
     <width>501</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout">
    <item>
     <widget class="QLabel" name="tripLabel">
      <property name="styleSheet">
       <string notr="true">font: 75 13pt &quot;Adobe Arabic&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="emissionReductionLabel">
      <property name="styleSheet">
       <string notr="true">font: 75 13pt &quot;Adobe Arabic&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="consumeCalorieLabel">
      <property name="styleSheet">
       <string notr="true">font: 13pt &quot;Adobe Arabic&quot;;
color: rgb(255, 0, 0);</string>
      </property>
      <property name="text">
       <string>20</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="horizontalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>150</y>
     <width>501</width>
     <height>51</height>
    </rect>
   </property>
   <layout class="QHBoxLayout" name="horizontalLayout_3">
    <item>
     <widget class="QLabel" name="label_6">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>里程(公里)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_7">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>减排(千克)</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="label_5">
      <property name="styleSheet">
       <string notr="true">font: 75 12pt &quot;Adobe Arabic&quot;;</string>
      </property>
      <property name="text">
       <string>消耗(卡)</string>
      </property>
     </widget>
    </item>
    <item>
     <spacer name="horizontalSpacer_2">
      <property name="orientation">
       <enum>Qt::Horizontal</enum>
      </property>
      <property name="sizeType">
       <enum>QSizePolicy::Minimum</enum>
      </property>
      <property name="sizeHint" stdset="0">
       <size>
        <width>40</width>
        <height>20</height>
       </size>
      </property>
     </spacer>
    </item>
   </layout>
  </widget>
  <widget class="QTableWidget" name="tableWidget">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>230</y>
     <width>510</width>
     <height>551</height>
    </rect>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
